cvat-ui/src/reducers/notifications-reducer.ts

```diff
@@ -390,26 +397,69 @@ export default function (state = defaultState, action: AnyAction): Notifications
                 },
             };
         }
-        case AuthActionTypes.ACCEPT_INVITATION_SUCCESS: {
+        case InvitationsActionTypes.GET_INVITATIONS_SUCCESS: {
+            if (action.payload.showNotification) {
+                return {
+                    ...state,
+                    messages: {
+                        ...state.messages,
+                        invitations: {
+                            ...state.messages.invitations,
+                            newInvitations: 'You have new organization invitations. [Click here](/invitations) to view them.',
+                        },
+                    },
+                };
+            }
+            return state;
+        }
+        case InvitationsActionTypes.ACCEPT_INVITATION_SUCCESS: {
+            return {
+                ...state,
+                messages: {
+                    ...state.messages,
+                    invitations: {
+                        ...state.messages.invitations,
+                        acceptInvitationDone: 'Invitation accepted successfully.',
+                    },
+                },
+            };
+        }
+        case InvitationsActionTypes.REJECT_INVITATION_SUCCESS: {
```

Comment:
I do not really think we need notifications when invitations accepted/rejected. 
It is quite natural that it was successful if errors not appeared (if only it is not long process, like exporting a dataset)
Of course we need some kind UI indication, but probably we already have on invitations page. 

Up to you, just my opinion here

Reply:
Removed them.